src/averageAges.js

```diff
@@ -15,11 +27,12 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const ageOfMen = people
+    .filter(({ died, sex }) => sex === MALE_SEX
+      && (century ? Math.ceil(died / 100) === century : true))
+    .map(({ born, died }) => died - born);
```

Comment:
Now everything is written correctly, but it is so difficult to read and expand. Break it into smaller parts.